CHANGELOG/CHANGELOG-0.3.md

```diff
@@ -0,0 +1,34 @@
+# Changelog since v0.2.0
+
+## Changes by Kind
+
+### Feature
+- Upgrade driver to use CSI spec 1.3.0 ([#45](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/pull/45), [@saikat-royc](https://github.com/saikat-royc))
+- Add support for Volume Expansion for Filestore CSI driver ([#55](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/pull/55), [@saikat-royc](https://github.com/saikat-royc))
+- Enable Backup Create/Delete  and Create Volume for a Backup source for CSI Filestore driver ([#61](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/pull/61), [@saikat-royc](https://github.com/saikat-royc))
```

Comment:
This is used for the CSI snapshot api, right? Might want to put that in there so it's easy to search for "snapshot" changes.

Reply:
Done